pkgs/applications/editors/spacevim/default.nix

```diff
@@ -0,0 +1,60 @@
+{ makeWrapper, vim_configurable, vimPlugins, fetchFromGitHub, writeTextDir
+, stdenv, spacevim_toml ? "" }:
+with stdenv;
+let
+  # At the moment, there is no way to write TOML given a attrset
+  # so the solution here is to allow a custom TOML file to be provided.
+  # The pkg includes a sensible TOML file for those that don't specify one.
+  config = if builtins.stringLength spacevim_toml == 0 then
+    "${builtins.readFile ./init.toml}"
+  else
+    spacevim_toml;
+  vim-customized = vim_configurable.customize {
+    name = "vim";
+    # Not clear at the moment how to import plugins such that
+    # SpaceVim finds them and does not auto download them to
+    # ~/.cache/vimfiles/repos
+    vimrcConfig.packages.myVimPackage = with vimPlugins; { start = [ ]; };
+  };
+  spacevimdir = writeTextDir "/init.toml" config;
+
+  spacevim = mkDerivation {
```

Comment:
Redundant `let` binding, just do `let ... in mkDerivation { ... }` instead

Reply:
Fixed